pkg/virtual/workspaces/registry/rest.go

```diff
@@ -356,6 +425,8 @@ func (s *REST) Delete(ctx context.Context, name string, deleteValidation rest.Va
 	if kerrors.IsNotFound(errorToReturn) {
 		errorToReturn = kerrors.NewNotFound(tenancyv1beta1.Resource("workspaces"), name)
 	}
+
+	// TODO(sttts): remove when GC is implemented
```

Comment:
Propagated TODO to https://github.com/kcp-dev/kcp/pull/2113 in the event it ends up being accepted.